src/main/java/spireMapOverhaul/zones/candyland/CandyLand.java

```diff
@@ -0,0 +1,197 @@
+                    rewardItem.relic = new Strawberry();
+                    rewardItem.text = rewardItem.relic.name;
+                    break;
+                default:
+                    SpireAnniversary6Mod.logger.info("Non-standard rarity found, replacing " + rewardItem.relic.name + " with Waffle");
+                    rewardItem.relic = new Waffle();
+                    rewardItem.text = rewardItem.relic.name;
+                    break;
+            }
+        } else if(rewardItem.type == RewardItem.RewardType.POTION){
+            while(rewardItem.potion.isThrown){
+                SpireAnniversary6Mod.logger.info("Try to replace " + rewardItem.potion.name + "with drinkable potion.");
+                rewardItem.potion = AbstractDungeon.returnRandomPotion(rewardItem.potion.rarity, false);
+            }
+        }
+    }
+
+    @Override
+    public void modifyRewards(ArrayList<RewardItem> rewards) {
+        SpireAnniversary6Mod.logger.info("Turning Gold into Chocolate");
+        for(RewardItem r : (ArrayList<RewardItem>) rewards.clone()){
+            if(r.type == RewardItem.RewardType.GOLD){
+                rewards.add(new HealReward("ChocolateCoins.png", r.goldAmt));
+            }
+        }
+        rewards.removeIf(r -> r.type == RewardItem.RewardType.GOLD);
+    }
+
+    public void postAddButtons(ArrayList<AbstractCampfireOption> buttons) {
+        for (AbstractCampfireOption button : buttons){
+            if (button instanceof SmithOption){
+                button.usable = false;
+            }
+        }
+    }
+
+    public void postUseCampfireOption(AbstractCampfireOption option) {
+        if(option instanceof RestOption) {
+            AbstractDungeon.player.increaseMaxHp(5, true);
+        }
+    }
+
+    public void postCreateShopCards(ArrayList<AbstractCard> coloredCards, ArrayList<AbstractCard> colorlessCards) {
+        ArrayList<AbstractCard> topRow = new ArrayList<>();
+        for(AbstractCard card : coloredCards){
+            ArrayList<AbstractConsumable> consumables = getConsumables();
+            consumables.removeIf(c -> c.rarity != card.rarity || c.type != card.type || containCard(topRow, c));
+            if(consumables.isEmpty()){
+                consumables = getConsumables();
+                consumables.removeIf(c -> c.type != card.type || containCard(topRow, c));
+
+            }
+            topRow.add(consumables.get(AbstractDungeon.cardRng.random(0, consumables.size()-1)));
+        }
+        coloredCards.clear();
+        coloredCards.addAll(topRow);
+
+        colorlessCards.clear();
+        colorlessCards.add(new Bite());
+        colorlessCards.add(new Feed());
+    }
+
+    public float modifyCardBaseCost(AbstractCard c, float baseCost) {
+        switch(c.cardID) {
+            case Bite.ID: return 80 + AbstractDungeon.merchantRng.random(1, 19); // Uncommon Colorless cost
+            case Feed.ID: return baseCost;
+            default: return (float) (baseCost*0.5);
```

Comment:
This seems pretty reasonable to me. I am a bit worried that people might not want to buy `Fleeting` cards even at half price, but we can see what people's feedback for this zone is.

The easiest cards to evaluate for me are the ones that give permanent benefits (Banana, Chocolate Bar, Cocktail), and the price for those doesn't seem too far off. Roughly 75 gold for 2 (3) max HP, 4 (5) regen, or a random drinkable potion. That might be a bit too much given the comparison to Strawberry and just buying potions, so maybe it makes sense to start out with 40% of the normal cost instead of 50%. That would take these down to 60 gold.

 are all worth considering at this price, at least. ~75 gold for 2 (3) max HP, 4 (5) regen, or a random drinkable potion.